### ISSUE TITLE
Updated catalyst flow, payment

### ISSUE THREAD

**Issue by CarrieTHA** [wileyandco/thehumanarray#32]
Organization is not my strong suit right now...and I am working on it because I feel I need it desperately at the moment to feel that everything is accounted for. 

In the meantime...here are the links to the new plans: 

Catalyst membership | $111 one-time set-up fee, includes profile space only, no program hosting: https://community.thehumanarray.com/users/onboarding/choose_plan?plan_id=608909&bundle_token=f437ac363b436f8c065ae84d9613de1d&utm_source=manual

Fuel to Catalyze membership +1| $222 one-time set-up fee, includes 1 paid program space:
https://community.thehumanarray.com/users/onboarding/choose_plan?plan_id=608913&bundle_token=f32313dd6e75a9b754e92fe5a20a517a&utm_source=manual

Fuel to Catalyze membership +2 | $333 one-time set-up fee, includes 2 paid program spaces:
https://community.thehumanarray.com/users/onboarding/choose_plan?plan_id=608915&bundle_token=83245d8dda3d8961d60d927f9fbbab7a&utm_source=manual

Fuel to Catalyze membership +3 | $444 one-time set-up fee, includes 2 paid program spaces:
https://community.thehumanarray.com/users/onboarding/choose_plan?plan_id=608917&bundle_token=379ade103cc0d7e9c5e95fed59801569&utm_source=manual

My thought is that they would choose how many programs they would want to host or if they just want to be in the community so that they can just get the link based on that question rather than seeing all of the links. 

Ready to get started? 
How many programs would you like to host on The Human Array? 
<dropdown>
0
1
2
3

Selection exposes the link for the plan that they select. 

Click to get started. 

I am working through the flow in my head and I think it is going to make sense like this: 

1. /Catalyst page has all the "sales" info of what a Catalyst is, what the membership includes, and the pricing information / how it works
2. Interested? Apply (somewhere we need to articulate what the process is to become a Catalyst (1. Fill out application 2. schedule a call 3. Approval)
3. Approve / Decline (have yet to think through the decline process - perhaps push to Seeker/Inquirer?)
4. For approved - send them the Intake Form /catalyst-intake-form
5. Once they have completed the form, the end page sends them to the page where they officially join/pay (want to look into if we can edit the link based on the answer to how many spaces - also thinking through their ability to still change that in case it isn't accurate)
6. As they join we currently have questions - with this new flow, we may no longer need the questions, but instead just an approval - or perhaps the approval can go away as well. The thought to consider here is if they land on the MN generated page of community.thehumanarray.com which shows all of the plans. We could modify that copy to include a link back to the Catalyst flow or an email if we don't want to show the actual plans there for Catalysts (aka keep the plans hidden)
7. When they join (aka pay) it triggers Adia to do the following: 

1. Adia sends welcome message in THA
2. Adia sends welcome email (I think we can automate this, but need to confirm)
3. Adia makes profile space + any paid program spaces
4. Adia creates Catalyst lists in Hubspot
5. Adia creates Zaps to connect Catalyst spaces to Hubspot
6. Adia sends email to Catalyst to review spaces

We are still learning through the process of how to support them / collect content for the actual programs. I have multiple Catalysts that are playing with me on this, so this part is still a WIP. 

The other piece that I want to start to layer into this is the branding piece. In the form, they are answering that question, so now I want to figure out how we support them in that, what they are looking for, etc. I will add this in a separate issue since I went WAY overboard here. LOL

**Comment by wileyandco** [comment]
Not overkill at all lol. Let’s talk through it at one, but generally sounds
good to me.




*Jeannette Wiley Detwiler*
Owner & Designer

wileyand.co
***@***.***
linkedin.com/in/jeannettewiley
cell: 315-283-0648 <315-283-0648>

**Comment by wileyandco** [comment]
@CarrieTHA https://www.thehumanarray.com/catalyst-learn-more-2024 Is up with a start of organizing this new process. Right now it's a section from the about us page that I think could work here or add value when distilled in a different way. I wanted to get the ideas out there and help to flush out how it works before going hard with new icons or look here. I think the pricing can live just on the next page, but tell me what you think. Hoping this helps!

**Comment by wileyandco** [comment]
@CarrieTHA The schedule a time on this should have times too? And should we use jotform for this too to keep it consistent? https://share.hsforms.com/1lDJSDc3JR12nbwidRiyTSwnv8cj

**Comment by wileyandco** [comment]
@CarrieTHA Ok the pricing page is set up with the filtering you requested. It works with the CMS function (good thing we set that up for the resources page - I love when it comes back around). So you have to click in and change the text within the excel-type thing in there for the repeater elements. I can't get the buttons to work right now and need to log off, but wanted to send your way for copy and functionality thoughts. Also, hopefully again this helps things feel closer to realized and less floaty :)

**Comment by wileyandco** [comment]
@CarrieTHA Realizing that right now each start now button will go to the form anyway and then I can make four separate checkout pages with one on each and a regular button with the correct link, correct?

**Comment by wileyandco** [comment]
@CarrieTHA These landing pages are all updated with button links working. I created one extra one named fuel-all to upsell from the regular catalyst membership page that has all the fuels on it. I think the individual ones are still valid because you don't want someone to be able to easily back out of a higher plan after already choosing it. I also added "limited time offer" to all of these. One more thing to think through is the messaging on these pages for when the monthly membership comes in. Something like "This fee is in addition to your monthly membership cost, which is calculated based on participation and helping the community thrive (calculated separately to keep it simple)." 

https://www.thehumanarray.com/catalyst-learn-more-2024

https://www.thehumanarray.com/catalyst-prices

https://www.thehumanarray.com/catalyst-payment-regular

https://www.thehumanarray.com/catalyst-payment-fuel-all

https://www.thehumanarray.com/catalyst-payment-fuel-1

https://www.thehumanarray.com/catalyst-payment-fuel-2

https://www.thehumanarray.com/catalyst-payment-fuel-3

I think we could also add after the "existing branding" sentence, a simple link to a "need branding?" landing and checkout page too, once that is ready. Or maybe that's on the thank you page or gets zapped in an email too. People may change their mind after they complete the form or when they get into the community and see how nice everyone else's are. We'd need a stripe link or something at least to make that one live. 

Adding simple branding page setup and make forms consistent to my list.

**Comment by CarrieTHA** [comment]
Does it make sense to have two pricing pages? One that is exposed where the
links still all lead to the application and then another that we use for
Adia? Or maybe we don't even need the one for Adia. If we are keeping the
application as the first step no matter what, then the pricing links would
still all go to the application except for the one that they are choosing
based on the number of programs. Help me think through this LOL I just
don't want them to be able to get to just joining without the application
until we are ready for that A/B test.


*Carrie Allen*
Founder
thehumanarray.com
IG: @thehumanarray

**Comment by CarrieTHA** [comment]
For the Catalyst flow, I just popped into Jotform and see that we can
change the landing page based on their answers to questions. (see
screenshot attached).

You're able to access Jotform, right? Would you be able to work on this
flow so that they all lead to the right places? The only thing is that I
first ask if they want to host (a previous question than the one in the
screenshot) and if they say no, this question doesn't appear. So I think
that we can set up a condition for that for just the basic profile space
($111) plan page to appear.

Does this make sense? Then we don't need Adia to send anything if this
flows correctly. Because either someone finds us, applies and then we sent
this form when they are approved. Or if they are a referral they go
straight to the intake form and then they get the "checkout" plan page as
the next step based on what they answered. Streamlining baby!


*Carrie Allen*
Founder
thehumanarray.com
IG: @thehumanarray

**Comment by wileyandco** [comment]
Right now it goes:
Catalyst learn more page
- one link goes to pricing page. All the links on this page just go to the
jotform to get approved.
The hidden pages (which I thought were going to adia to be set up in the
emails) we could connect from a jotform or approval, have similar look but
their buttons actually go to the payment links.
- one of these is the plain $111. It has a button which sends you to one
with payment links for all the fuel ones in case they want to upgrade.

I can access jotform, just have to remember how.




*Jeannette Wiley Detwiler*
Owner & Designer

wileyand.co
***@***.***
linkedin.com/in/jeannettewiley
cell: 315-283-0648 <315-283-0648>

**Comment by wileyandco** [comment]
@CarrieTHA The long-form application jotform is updated to the correct checkout pages, based on their selections! Just texted it and it works. 

![Image](https://github.com/wileyandco/thehumanarray/assets/78058722/4d0887e1-3899-486d-aac8-17b8523c96df)

**Comment by CarrieTHA** [comment]
YAY!!!


*Carrie Allen*
Founder
thehumanarray.com
IG: @thehumanarray

**Comment by CarrieTHA** [comment]
And...as soon as we finish we're ready to update! 

Not sure if you want to start a new issue, but wanted to get us started. 

I think since we're doing subscription + set up fees, we need to manage payment outside of MN. You can only buy one plan at a time. 

I'll send you the details for my Stripe account so we can figure out the mechanics. 

So here's what I believe will need to happen. 

1. Same application flow
2. Intake form lands on appropriate checkout page
3. Stripe manages transaction of subscription+fee
4. After payment processes, then guided to create account (MN link) - can also be sent via email

What this would also mean is that we can likely figure out a referral / reward program through Stripe + they'll have coupon codes if we want to give referral rewards / discounts. At least I believe! I can't help myself, so I'll do some poking around as well. Not much else should need to change other than how we process the payment and moving them to create an account after payment.

**Comment by wileyandco** [comment]
Sounds good, thanks for the heads up! No worries knew this was coming and
glad we’re here 🌞
